packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -509,7 +511,7 @@ const CancellationDialogMessage = ({
           Have questions? Visit <a href={supportFormUrl}>Mozilla Support</a>.
         </p>
       </Localized>
-    </DialogMessage>
+    </Modal>
   );
 };
 
```

Comment:
Commenting on the change below, for moving the modal's `en-US.ftl` into `fxa-react`: this won't work since we don't have l10n set up in `fxa-react`, we have [a spike](https://mozilla-hub.atlassian.net/browse/FXA-4477) for it that Dan and I are hoping to work through but for now we're going to need to leave that FTL file where it's being used (looks like payments). You might need to rename it to something other than `index` temporarily but that's fine since our ftl merging script looks for `*.ftl` files, but we shouldn't have any FTL files in fxa-react right now. It would also be nice if you could add a comment on that spike noting where it's located so when we _do_ localize `fxa-react` we can move it where it should ultimately end up being.